packages/apollo-codegen-swift/src/__tests__/__snapshots__/codeGeneration.ts.snap

```diff
@@ -2,8 +2,14 @@
 
 exports[`Swift code generation #classDeclarationForOperation() should generate a class declaration for a mutation with variables 1`] = `
 "public final class CreateReviewMutation: GraphQLMutation {
+  /// mutation CreateReview($episode: Episode) {
+  ///   createReview(episode: $episode, review: {stars: 5, commentary: \\"Wow!\\"}) {
+  ///     stars
+  ///     commentary
+  ///   }
+  /// }
```

Comment:
To be honest, I very explicitly *do not want this*. This comment just bloats the file. I notice all your tests are relatively short snippets, but for comparison, I have a single query I'm looking at right now that spans 56 lines, and I'd really not like to have a 56-line comment in the generated source obscuring the actual datatype.

Reply:
There is at least one other person who has [explicitly requested multiline string support](https://github.com/apollographql/apollo-ios/issues/179). 

In your opinion, what is better when we're talking about things of similar length: A multiline string that has to be compiled or a multiline comment that doesn't?

Reply:
The comment approach is better. Can we make it an option as to whether to have it though?